CHANGELOG.md

```diff
@@ -6,14 +6,71 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 ## Upcoming release
 
-### Added
+nothing yet
 
-* Support for Bazel 2.0.0.
+### [0.12.0] - 2020-03-DD TODO set day
 
-### Changed
+## Highlights
+
+* Various improvements to Windows support.
+
+* Support for Bazel 2.0.0
+
+* Minimum supported Bazel version is now 0.29
```

Comment:
At least on MacOS with GHC bindist Bazel 0.29 is failing, see [this CircleCI run](https://circleci.com/gh/tweag/rules_haskell/8684?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link).

Reply:
I've modified the _Highlights_ section accordingly in cb537c2.